admin/css/login.css

```diff
@@ -0,0 +1,121 @@
+.text-primary {
+  color: #204a87ff !important;
+}
+
+body,
+html {
+  font-family: "URW Gothic L" !important;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+}
+
+.container-fluid {
```

Comment:
Codacy found an issue: [Missing vendor-prefixed CSS gradients for Firefox 3.6+, Webkit (Safari 5+, Chrome), Opera 11.1+.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)